operator/controllers/btpoperator_controller.go

```diff
@@ -660,14 +658,14 @@ func (r *BtpOperatorReconciler) handleHardDelete(ctx context.Context, namespaces
 			return
 		}
 
-		time.Sleep(r.hardDeleteTimeout / 20)
+		time.Sleep(HardDeleteTimeout / 20)
```

Comment:
We can use var from L64 here to have configurable interval for hard delete check